apps/frontend/src/app/3_organisms/FastBtcDialog/constants.ts

```diff
@@ -0,0 +1,13 @@
+// https://github.com/DistributedCollective/FastBTC/blob/development/config/config-main.js#L15
+export const DEPOSIT_FEE_SATS = 6000;
```

Comment:
Noticed that this is actually 5000 in the linked file, please double check with Rainer what correct value should be here

Reply:
Good catch, I'll ask Rainer/Hantti.

Reply:
Hantti confirmed it should be 5000 so I adjusted it.